integrations/amazon_bedrock/tests/test_generator.py

```diff
@@ -259,11 +258,10 @@ def test_prepare_body_with_default_params(self) -> None:
         assert body == expected_body
 
     def test_prepare_body_with_custom_inference_params(self) -> None:
-        layer = AnthropicClaudeAdapter(model_kwargs={}, max_length=99)
+        layer = AnthropicClaudeAdapter(model_kwargs={})
         prompt = "Hello, how are you?"
         expected_body = {
             "messages": [{"role": "user", "content": "Hello, how are you?"}],
-            "max_tokens": 50,
```

Comment:
I don't think this is needed to be removed, it's unrelated to the changes right?